src/azure-cli/setup.py

```diff
@@ -30,175 +30,179 @@
+    description="Microsoft Azure Command-Line Tools",
     long_description=README,
-    license='MIT',
-    author='Microsoft Corporation',
-    author_email='azpycli@microsoft.com',
-    url='https://github.com/Azure/azure-cli',
+    license="MIT",
+    author="Microsoft Corporation",
+    author_email="azpycli@microsoft.com",
+    url="https://github.com/Azure/azure-cli",
     zip_safe=False,
     classifiers=CLASSIFIERS,
     scripts=[
-        'az',
-        'az.completion.sh',
-        'az.bat',
+        "az",
+        "az.completion.sh",
+        "az.bat",
     ],
-    packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests", "azure", "azure.cli"]),
+    packages=find_packages(
+        exclude=["*.tests", "*.tests.*", "tests.*", "tests", "azure", "azure.cli"]
+    ),
     install_requires=DEPENDENCIES,
-    python_requires='>=3.6.0',
+    python_requires=">=3.6.0",
     package_data={
-        'azure.cli.command_modules.acr': ['*.json'],
-        'azure.cli.command_modules.botservice': ['*.json', '*.config'],
-        'azure.cli.command_modules.monitor.operations': ['autoscale-parameters-template.json'],
-        'azure.cli.command_modules.servicefabric': [
-            'template/windows/template.json',
-            'template/windows/parameter.json',
-            'template/linux/template.json',
-            'template/linux/parameter.json',
-            'template/service/template.json',
-            'template/service/parameter.json'
+        "azure.cli.command_modules.acr": ["*.json"],
+        "azure.cli.command_modules.botservice": ["*.json", "*.config"],
+        "azure.cli.command_modules.monitor.operations": [
+            "autoscale-parameters-template.json"
         ],
-        'azure.cli.command_modules.appservice': [
-            'resources/WebappRuntimeStacks.json',
-            'resources/GenerateRandomAppNames.json'
+        "azure.cli.command_modules.servicefabric": [
+            "template/windows/template.json",
+            "template/windows/parameter.json",
+            "template/linux/template.json",
+            "template/linux/parameter.json",
+            "template/service/template.json",
+            "template/service/parameter.json",
+        ],
+        "azure.cli.command_modules.appservice": [
+            "resources/WebappRuntimeStacks.json",
+            "resources/GenerateRandomAppNames.json",
+        ],
+        "azure.cli.command_modules.rdbms": [
+            "randomname/adjectives.txt",
+            "randomname/nouns.txt",
+            "templates/mysql_githubaction_template.yaml",
+            "templates/postgresql_githubaction_template.yaml",
         ],
-        'azure.cli.command_modules.rdbms': [
-            'randomname/adjectives.txt',
-            'randomname/nouns.txt',
-            'templates/mysql_githubaction_template.yaml',
-            'templates/postgresql_githubaction_template.yaml'
-        ]
     },
-    cmdclass=cmdclass
+    cmdclass=cmdclass,
```

Comment:
Adding this comma `,` is unnecessary.

Reply:
Oof, vscode did that unexpectedly. I'll correct the PR.